### PR TITLE
Adding information about the PUT method for AWS integration.

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Adds info about the PUT method for the AWS create integration API for the new Update AWS integration logic

### Motivation
It's now released.

### Preview link

* https://docs-staging.datadoghq.com/gus/aws-put/api/?lang=python#integration-aws
* https://docs-staging.datadoghq.com/gus/aws-put/api/?lang=python#update-an-aws-integration

### CODE REVIEW

**Comment by DanielLanger** [comment]
Do we not want this under a separate "Update an AWS integration" section?

File: content/en/api/integrations_aws/code_snippets/aws_update.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "account_id": "<AWS_ACCOUNT_ID>",
+        "filter_tags": ["<KEY>:<VALUE>"],
+        "host_tags": ["<KEY>:<VALUE>"],
+        "role_name": "DatadogAWSIntegrationRole",
+        "account_specific_namespace_rules": {
+        	"auto_scaling": false,
+        	"opsworks": false
+        }
+    }'\
+"https://api.datadoghq.com/api/v1/integration/aws?api_key=${api_key}&application_key=${app_key}"
```
Comment (nmuesch) [inline content/en/api/integrations_aws/code_snippets/aws_update.sh]: This endpoint requires query parameters to know which account to update, 
`/api/v1/integration/aws?account_id=xyz&role_name=abc&api_key=1234&app_key=5678`

Comment (l0k0ms) [reply content/en/api/integrations_aws/code_snippets/aws_update.sh]: fixed :) thanks for the feedback 👍

File: content/en/api/integrations_aws/aws_update.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+external_redirect: /api/#update-an-aws-integration
+---
+
+## Update an AWS integration
+
+Update a Datadog-Amazon Web Services integration.
+
+**Note**: Using the `PUT` method updates your integration configuration by **replacing** your current configuration with the new one sent to your Datadog organization.
+
+##### ARGUMENTS
```
Comment (nmuesch) [inline content/en/api/integrations_aws/aws_update.md]: This looks like it needs the query params for account_id and role_name too

File: content/en/api/integrations_aws/code_snippets/result.aws_update.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "external_id": "12345678910111213"
```
Comment (nmuesch) [inline content/en/api/integrations_aws/code_snippets/result.aws_update.sh]: I believe this doesn't return the external id and instead just returns an empty response `{}`

**Pull request merged**